website/versioned_docs/version-2.0/schema-schema.md

```diff
@@ -47,9 +47,20 @@ type Mutation {
 ## Scalar Types
 
 Scalar types are custom types that require a `directive` (which we'll learn about later) to serialize, deserialize and validate values.
+<br><br>
+Ligthouse includes a DateTime scalar type. This scalar type can be used to handle Carbon DateTime fields. 
+You can use it inside your schema like this:
 
 ```graphql
-scalar DateTime @scalar(class: "DateTimeScalar")
+scalar DateTime @scalar(class: "App\\GraphQL\\Scalars\\DateTime")
```

Comment:
The path to the class does not seem right. Also, i think we should dedicate a seperate section to our custom scalars. While atm we only have one, i think it would be great to add some common ones to Lighthouse.

Reply:
Good point. I actually copied it from the `directives-schema` file, but I hadn't noticed that it was not the actual class from the package being referenced there. I updated it to match the included file in my explanation.

I also moved the explanation to a seperate section, the class is also included in 2.0 so it seemed appropriate to add it for that version as well.